spec/filters/geoip_spec.rb

```diff
@@ -93,8 +93,8 @@
 
       it "should set other subfields of 'target' properly" do
         expect(event.get("target").to_hash.keys.sort).to eq(["city_name", "ip", "region_name"])
-        expect(event.get("[target][city_name]")).to eq("Salem")
-        expect(event.get("[target][region_name]")).to eq("New Hampshire")
+        expect(event.get("[target][city_name]")).to eq("Watertown")
+        expect(event.get("[target][region_name]")).to eq("Massachusetts")
```

Comment:
This looks like it just due to an update to the DB ... the two locations are kinda close:
<img width="1170" alt="image" src="https://user-images.githubusercontent.com/976291/34234768-bf505320-e5b2-11e7-8d5a-b1f86f45fb4a.png">